code/modules/mob/new_player/sprite_accessories.dm

```diff
@@ -1292,6 +1292,12 @@
 	name = "Rough-Cropped Mane"
 	icon_state = "rough"
 
+/datum/sprite_accessory/hair/vulpkanin/vulp_hair_raine
+	name = "Raine"
+	icon_state = "vulp_hair_raine"
+	species_allowed = list("Vulpkanin")

```

Comment:
Redundant, declared already in parent

Reply:
Hey, we worked hard to eliminate unnecessary `species_allowed`! :wink:

Reply:
That species_allowed list annoyed me for the longest time. When was it fixed? I can't remember

Reply:
It was done in this mega-PR. Should definitely not be used as an example of atomic development :innocent: https://github.com/ParadiseSS13/Paradise/pull/5416